src/V3Coverage.cpp

```diff
@@ -404,6 +434,8 @@ class CoverageVisitor final : public VNVisitor {
     // VISITORS - LINE COVERAGE
     // Note not AstNodeIf; other types don't get covered
     void visit(AstIf* nodep) override {
+        if (nodep->user2()) { return; }
+
```

Comment:
```suggestion
        if (nodep->user2()) return; 
```

Similar other places.  (I realize you probably had multilines that got squashed.)